credential/base/src/main/java/org/wildfly/security/keystore/PasswordKeyStoreSpi.java

```diff
@@ -91,7 +91,9 @@ public void engineSetEntry(final String alias, final KeyStore.Entry entry, final
         if (protParam != null) {
             throw log.keyCannotBeProtected(alias);
         }
-        HashMap<String, PasswordEntry> map, newMap;
+        HashMap<String, PasswordEntry> map;
+        HashMap<String,PasswordEntry> newMap;
```

Comment:
@nasonawa Justa t total minor, can you please fix the formatting by adding a space before the `PasswordEntry` of newMap initialization? Thank you!